agb/examples/sprites.rs

```diff
@@ -100,6 +101,19 @@ fn all_tags(gfx: &ObjectController) {
 fn main(mut gba: agb::Gba) -> ! {
     let gfx = gba.display.object.get();
 
+    let timers = gba.timers.timers();
+    let mut my_timer = timers.timer0;
+    my_timer.set_interrupt(true);
+    my_timer.set_overflow_amount(10000);
+    my_timer.set_enabled(true);
+
+    agb::add_interrupt_handler!(
+        agb::interrupt::Interrupt::Timer0,
+        |_key: &CriticalSection| {
+            agb::println!("{:#010x}", agb::get_program_counter_before_interrupt());
+        }
+    );
```

Comment:
Is it possible to convert this into `profile!(timers.timer0)` or something? Happy to add a method to timer which returns the `agb::interrupt::Interrupt` to support this